packages/spruce-types/src/gql/types/SprucebotAvatar.gql

```diff
@@ -0,0 +1,32 @@
+"Sprucebot's current state of mind"
+enum SprucebotAvatarStateOfMind {
+	"For when nothing in particular is happening"
+	chilling
+	"When submitting a form or running some other operation"
+	thinking
+	"Asking a question or leaning new things"
+	inquisiting
+	"Something has finished and we're so excited!"
+	jamming
+}
+
+enum SprucebotAvatarSize {
+	"Inline with text"
```

Comment:
Can these be more prescriptive/documentation-focused, i.e. "This size should be used inline with text" "This size is best used for subheadings" etc